src/lib/pages/schedule-edit.svelte

```diff
@@ -0,0 +1,192 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+
+  import { submitEditSchedule, loading, error } from '$lib/stores/schedules';
+  import { routeForSchedule } from '$lib/utilities/route-for';
+
+  import Icon from '$lib/holocene/icon/icon.svelte';
+  import Button from '$lib/holocene/button.svelte';
+  import Loading from '$holocene/loading.svelte';
+  import Input from '$lib/holocene/input/input.svelte';
+  import { fetchSchedule } from '$lib/services/schedule-service';
+  import SchedulesCalendarView from '$lib/components/schedule/schedules-calendar-view.svelte';
+  import type { Schedule } from '$types';
+
+  let namespace = $page.params.namespace;
+  let scheduleId = $page.params.schedule;
+
+  let errors = {};
+  let name = '';
+  let workflowType = '';
+  let workflowId = '';
+  let taskQueue = '';
+  let daysOfWeek: string[] = [];
+  let daysOfMonth: number[] = [];
+  let months: string[] = [];
+  let days = '';
+  let hour = '';
+  let minute = '';
+  let second = '';
+  let phase = '';
```

Comment:
Again maybe `phase` should be more strongly typed since it can only be a set of pre-defined strings.